### PR TITLE
use Jest as test framework

### PR DESCRIPTION
**Pull request by AbdilFattahBoshi**
Hii @NourSammour  I have installed Jest and use it as test framework
Capture

### CODE REVIEW

**Comment by AbdilFattahBoshi** [comment]
some conflicts was occur and I solve it @NourSammour

File: package.json [hunk 1]
```diff
@@ -5,25 +5,24 @@
   "scripts": {
     "dev": "node server.js",
     "build": "next build",
-    "start": "cross-env NODE_ENV=production node server.js"
+    "start": "cross-env NODE_ENV=production node server.js",
+	  "test": "jest"
```
Comment (NourSammour) [inline package.json]: fix format issue

**Review by NourSammour — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (AbdilFattahBoshi) [reply package.json]: @NourSammour  I have fix format issue

File: package.json [hunk 2]
```diff
@@ -5,25 +5,24 @@
...
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6"
   },
-
   "devDependencies": {
     "cross-env": "^5.2.0",
     "babel-eslint": "^10.0.2",
     "eslint": "^6.1.0",
     "eslint-plugin-flowtype": "^3.12.1",
-    "eslint-plugin-react": "^7.14.2"
```
Comment (NourSammour) [inline package.json:23]: why did you remove `eslint-plugin-react` ?

**Review by NourSammour — CHANGES_REQUESTED** [review]

**Review by NourSammour — APPROVED** [review]
